deploy.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -e # Exit with nonzero exit code if anything fails
+
+TARGET_BRANCH="gh-pages"
+
+function doCompile {
+  GRADLE_OPTS='-Xmx600m -Dorg.gradle.jvmargs="-Xmx1500m"' ./gradlew release
```

Comment:
You can simply add a release target to the build matrix. So this step is not required.